### PR TITLE
fix: SQL REPL, SQLite module version bump and SQLite Readme update [Fixes VM-225]

### PR DESCRIPTION
**Pull request by raftedproc**

### CODE REVIEW

File: marine-examples/sqlite/README.md [hunk 1]
```diff
@@ -1,115 +1,112 @@
...
 
-As a result, our service is comprised of two modules: the SQLite Wasm module and our sqlite module. 
 
-## Build
+[SQLite](https://www.sqlite.org/index.html) is ubiquitous persistence solution also available in the Fluence stack. Unlike basic compute modules, using SQLite in a service is a little more intricate. In order to create a service, one needs two dependencies:
+- SQLite Wasm module that one can get either in a form of a [released module](https://github.com/fluencelabs/sqlite/releases/tag/v0.17.1_w) or build it from scratch following [this](https://github.com/fluencelabs/sqlite) readme.
+- SQLite connector that is [WASM IT compliant](https://crates.io/crates/marine-sqlite-connector) [sqlite](https://github.com/stainless-steel/sqlite) package implementation.
```
Comment (boneyard93501) [inline marine-examples/sqlite/README.md:11]: we should link to our connector [sqlite](https://github.com/fluencelabs/sqlite-wasm-connector)

File: marine-examples/sqlite/README.md [hunk 2]
```diff
@@ -1,115 +1,112 @@
...
 
-## Run
 
-To check how it works, we need to run `mrepl`:
 
+## How to use
+
+
+
+One needs to install marine and mrepl Rust crates to try this example. Here is a simple 'CREATE-INSERT-SELECT' scenario output using the artifacts of this repo.
```
Comment (boneyard93501) [inline marine-examples/sqlite/README.md:53]: > One needs to install marine and mrepl Rust crates to try this example
how about:
Here is a simple 'CREATE-INSERT-SELECT' scenario using the Wasm modules in the artifacts directory. To follow along, you need [marine](https://crates.io/crates/marine) and [mrepl](https://crates.io/crates/mrepl) installed.

File: marine-examples/sqlite/README.md [hunk 3]
```diff
@@ -1,115 +1,112 @@
...
+
+```bash
+$cargo install marine mrepl
```
Comment (boneyard93501) [inline marine-examples/sqlite/README.md:56]: this assumes you are in nightly. better make it explicit. also, we are currently constraint to nightly nightly-2022-12-06-x86_64-apple-darwin or older.

File: marine-examples/sqlite/README.md [hunk 4]
```diff
@@ -1,115 +1,112 @@
...
\ No newline at end of file
+
+## Support
+
+Please, [file an issue](https://github.com/fluencelabs/marine/issues) if you find a bug. You can also contact us at [Discord](https://discord.com/invite/5qSnPZKh7u) or [Telegram](https://t.me/fluence_project).  We will do our best to resolve the issue ASAP.
+
+
+## Contributing
+
+Any interested person is welcome to contribute to the project. Please, make sure you read and follow some basic [rules](./CONTRIBUTING.md).
```
Comment (boneyard93501) [inline marine-examples/sqlite/README.md:107]: dead link

File: marine-examples/sqlite/README.md [hunk 5]
```diff
@@ -1,115 +1,112 @@
...
+
+
+## License
+
+All software code is copyright (c) Fluence Labs, Inc. under the [Apache-2.0](./LICENSE) license.
```
Comment (boneyard93501) [inline marine-examples/sqlite/README.md:112]: dead link

File: marine-examples/sqlite/README.md (hunk 1 above)
Comment (raftedproc) [reply marine-examples/sqlite/README.md:11]: Good catch. I Will fix this.

File: marine-examples/sqlite/README.md (hunk 2 above)
Comment (raftedproc) [reply marine-examples/sqlite/README.md:53]: Sounds good to me.

File: marine-examples/sqlite/README.md (hunk 3 above)
Comment (raftedproc) [reply marine-examples/sqlite/README.md:56]: This is a very good point indeed.
@mikevoronov should we make this explicit with the current state of affairs, versions, modules?

File: marine-examples/sqlite/README.md (hunk 1 above)
Comment (raftedproc) [reply marine-examples/sqlite/README.md:11]: Fixed

File: marine-examples/sqlite/README.md (hunk 2 above)
Comment (raftedproc) [reply marine-examples/sqlite/README.md:53]: Fixed.

File: marine-examples/sqlite/README.md (hunk 5 above)
Comment (raftedproc) [reply marine-examples/sqlite/README.md:112]: Fixed

File: marine-examples/sqlite/README.md (hunk 4 above)
Comment (raftedproc) [reply marine-examples/sqlite/README.md:107]: Fixed

**Pull request merged**